covid-frontend/src/components/Answers/index.js

```diff
@@ -93,13 +100,13 @@ class Answers extends PureComponent {
 
         <Row gutter={24} className={styles.titleRow}>
           <Col span={24}>
-            <InputContainer label="Question" fluid>
+            <InputContainer label="Ihre Frage" fluid>
               <AutoComplete
                 className={styles.autocomplete}
                 size="large"
                 value={search.currentString}
                 defaultActiveFirstOption={false}
-                placeholder="Ask any question about Corona..."
+                placeholder="Stellen Sie eine Frage zu Covid-19 (Corona-Virus)"
```

Comment:
I am going to merge it for now, but I think we should add multi language options as next step instead of only having german.